packages/lib/src/components/internal/Await/Await.tsx

```diff
@@ -49,7 +49,7 @@ function Await(props: AwaitComponentProps) {
                 }
             };
             // Send success response to onAdditionalDetails
-            return props.onComplete(state, this);
+            return props.onComplete(state);
```

Comment:
Does removing 'this' imply that in the 'onAdditionalDetails' , the 'component' property is undefined?